gulpfile.js

```diff
@@ -1336,6 +1352,42 @@ gulp.task(
   })
 );
 
+gulp.task(
+  "typestest",
+  gulp.series(
+    "testing-pre",
+    "generic",
+    "types",
+
+    function () {
+      var packageJsonSrc = packageBowerJson()[0];
+      var TYPESTEST_DIR = BUILD_DIR + "typestest/";
+
+      return merge([
+        packageJsonSrc.pipe(gulp.dest(TYPESTEST_DIR)),
+        gulp
+          .src([
+            GENERIC_DIR + "build/pdf.js",
+            GENERIC_DIR + "build/pdf.worker.js",
+            SRC_DIR + "pdf.worker.entry.js",
+          ])
+          .pipe(gulp.dest(TYPESTEST_DIR + "build/")),
+        gulp
+          .src(TYPES_BUILD_DIR + "**/**")
+          .pipe(gulp.dest(TYPESTEST_DIR + "build/")),
+      ]);
+    },
```

Comment:
Also, defining this inline isn't really helping readability of the overall task, and it really ought to be moved into a helper function/task (e.g. `typestest-pre`) similar to what's done in other parts of the gulpfile.